.github/workflows/triage.yml

```
@@ -0,0 +1,53 @@
+name: Triage new issues
+
+on:
+  issues:
+    types: [ opened ]
+
+env:
+  GITHUB_TOKEN: ${{ secrets.PROJECT_BOARD_AUTOMATION }}
+  ORGANIZATION: ethereum
+  PROJECT_NUMBER: 43
+  DRY_RUN: true
+
+jobs:
+  triage_issues:
+    runs-on: ubuntu-latest
+    if: join(github.event.issue.labels) == ''
+    steps:
+      - name: Retrieve project data
+        env:
+          ORGANIZATION: ${{ env.ORGANIZATION }}
+          PROJECT_NUMBER: ${{ env.PROJECT_NUMBER }}
+        run: |
+          gh api graphql --raw-field query='
+            query($org: String!, $number: Int!) {
+              organization(login: $org) {
+                project(number: $number) {
+                  columns(first: 10) {
+                    nodes {
+                      id,
+                      name
+                    }
+                  }
+                }
+              }
+            }' --raw-field org=$ORGANIZATION --field number=$PROJECT_NUMBER >> project_data.json
+          echo 'COLUMN_ID='$(jq '.data.organization.project.columns.nodes[] | select(.name == "Triage") | .id' project_data.json) >> $GITHUB_ENV
+          echo 'COLUMN_NAME='$(jq '.data.organization.project.columns.nodes[] | select(.name == "Triage") | .name' project_data.json) >> $GITHUB_ENV
+      - name: Add issue#${{ github.event.issue.number }} to Triage column
+        env:
+          ISSUE_ID: ${{ github.event.issue.node_id }}
+          PROJECT_NUMBER: ${{ env.PROJECT_NUMBER }}
+        run: |
+          if [ ${{ env.DRY_RUN }} ]; then
```

You should compare with "true". Otherwise the condition will still be true when you change `DRY_RUN` to `false`.

Also, this is not a bash variable so it needs quotes even in `[[ ]]` in case it contains spaces.
```suggestion
          if [[ "${{ env.DRY_RUN }}" == true ]]; then
```

---

Indeed. The true should also be between `"` I believe.

---

Technically yes (without quotes it's a regex) but since it does not have any special chars, it's safe not to quote it here.